pkg/arvo/app/landscape/index.html

```diff
@@ -24,6 +24,5 @@
     <script src="/~landscape/js/channel.js"></script>
     <script src="/~landscape/js/session.js"></script>
     <script src="/~landscape/js/bundle/index.f58fbbc4b037bb976a2a.js"></script>
-    <script src="https://sdk.amazonaws.com/js/aws-sdk-2.1.12.min.js"></script>
```

Comment:
I have no idea how or why this got dropped, but I restored it in next-userspace.